src/modules/guide-steps.js

```diff
@@ -4,8 +4,17 @@ let currentStep = 1;
 
 const smallScreenWidth = 640;
 
+export function maybeGoToNextStep() {
```

Comment:
It would be better this function is not here. Can you create a function that returns currentStep value here and then let a function in `guide.js` determine if it is eligible for moving to the next step?